app/src/components/job/hr/CreateJob.jsx

```diff
@@ -18,9 +18,11 @@ export default function HRCreateJob() {
     // If detail key exists in response, there are schematic errors in the request
     if (res.detail && res.detail.length > 0) {
       const errorList = [];
-      res.detail.forEach((error) => {
-        errorList.push(error.msg);
-      }); 
+      if (res.detail.map) {
```

Comment:
I think we could check if res.detail is an array instead because if we check properties, what if res.detail is an object with the key "map".